### ISSUE TITLE
Island Expeditions Map

### ISSUE THREAD

**Issue by Palonblade** [mikepauer/Carbonite#121]
The Island Expeditions map is missing. Please let me know if there is any data I can get for you and if so how to do so.

**Comment by Palonblade** [comment]
Is there anything I can do to help with this?

**Comment by IrcDirk** [comment]
Hmmm... dont think so. I need UiMapIDs of thouse maps, will make those Expeditions then one after other i will add those. Be patient please, many other things are in line.

**Comment by Palonblade** [comment]
There are a total of 7 Islands available that players may visit during Island Expeditions. There is potential for more Islands to be added in future patches.
Molten Cay
Dread Chain
Whispering Reef
Skittering Hollow
Rotting Mire
Verdant Wilds
Ungol Ruins
Only 3 of these Islands will be available to visit throughout a week. The Islands available for the week are shown on the Expedition Map when queuing for Island Expeditions.

I just did Rotting Mire and there was no Carb map. I'm not sure if you just did one or attempted to do all

**Comment by IrcDirk** [comment]
Update Alpha release, i did the same map and it worked.

**Comment by Palonblade** [comment]
Strange, I had Twitch set to Alpha but the latest didn't show up yet. I downloaded manually and the map was there when I did Ungol Ruins :). The points of interest didn't show but having the map makes it so much better and thanks for all you do!

Points of Interest:
 Scouted Azerite: Location of some Azerite source that has yet to be revealed. This usually marks the location of a potential Powerful Foe or Bonus Objective. Not every objective will be marked this way however. When a member of your expedition team explores this area the icon will vanish.
 Azerite Source: Location of an interactable that awards Azerite. Usually a Chest, Satchel, or Mining Node containing Azerite.
 Powerful Foe: A Rare or Rare Elite enemy awarding a large amount of Azerite.
Bonus Objective: Location of a friendly NPC in need of help. The quest details are discovered when any member of your team walks up to the NPC and tracked on the Bonus Objectives quest tracker.
Enemy Faction Member: Location of a member of the enemy faction.
 Alliance Ship: Kul'Tiran ship, spawn location of Alliance members. Horde players too close will be bombarded by the ship cannons.
 Horde Ship: Zandalari ship, spawn location of Horde members. Alliance players too close will be bombarded by the ship cannons.
Portal: Location of a portal that will take you to another location.
